spec/gemstash/cli_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+
+RSpec.describe 'CLI executable', type: :aruba do
+  before do
+    run_command('exe/gemstash --version')
+  end
+
+  # Substring
```

Comment:
Is this comment necessary?

Reply:
Thanks for the sharp eye, I removed that and force-pushed.